1_historical_RS_data_collation.R

```diff
@@ -8,10 +8,15 @@ tar_source("1_historical_RS_data_collation/src/")
 # branches of the Landsat_C2_SRST repository: nw-poudre-historical and nw-er3z21-historical.
 # At this time, this is run outside of the {targets} workflow presented here. 
 
+# prep folder structure
+dir.create("1_historical_RS_data_collation/in/")
+dir.create("1_historical_RS_data_collation/mid/")
+dir.create("1_historical_RS_data_collation/out/")
+
 p1_targets_list <- list(
   # download the NW and CLP data from Google Drive
   tar_target(
-    name = p1_download_historical_NW_CLP,
+    name = p1_downloaded_historical_NW_CLP,
```

Comment:
p1... I believe p1 is a construct of something i worked on for AquaMatch. Definitely not necessary naming convention. Perhaps we can explore a better way to link these targets to their grouped target folder's name. for example: prefix all targets in t1_locs_poly_setup with t1.

Reply:
I absolutely agree! This is strongly related to the conversation we were having yesterday. Because this workflow will eventually have a bookdown, I'm going to transfer all lists to alpha-labels and make sure that the group names match across all files!